src/System.Drawing.Common/tests/Printing/PrintDocumentTests.cs

```diff
@@ -244,16 +244,32 @@ public void ToString_ReturnsExpected()
 
         private void AssertDefaultPageSettings(PageSettings pageSettings)
         {
-            Assert.Equal(new Rectangle(0, 0, 850, 1100), pageSettings.Bounds);
+            // A4 and Letter are both common default sizes for systems to have.
+            switch (pageSettings.PaperSize.Kind)
+            {
+                case PaperKind.A4:
+                    Assert.Equal(new Rectangle(0, 0, 827, 1169), pageSettings.Bounds);
+                    Assert.Equal(827, pageSettings.PrintableArea.Width, 0);
+                    Assert.Equal(1169, pageSettings.PrintableArea.Height, 0);
```

Comment:
Strictly A4 is 8.26771653543 × 11.6929133858. as it's defined as 210mm × 297mm. I assume no printer reports back with greater precision than hundreds of an inch, but if anyone finds this PR tracking down a mysterious imprecision of a fraction of a millimetre, this is why.